static/app/views/setupWizard/index.tsx

```diff
@@ -44,6 +55,11 @@ function SetupWizard({hash = false}: Props) {
     return () => window.clearInterval(pollingInterval);
   }, [checkFinished]);
 
+  const urlParams = new URLSearchParams(location.search);
+  const projectPlatform = urlParams.get('project_platform');
+  const docsLink =
+    platformDocsMapping[projectPlatform || ''] || 'https://docs.sentry.io/';
```

Comment:
shouldn't we use the router for this?

Reply:
If we don't have a router available then let's put this into a state that never updates or a use memo so that the link predictably stays the same.

```suggestion
  const docsLink = useMemo(() => {
    const urlParams = new URLSearchParams(location.search);
    const projectPlatform = urlParams.get('project_platform');
    return platformDocsMapping[projectPlatform || ''] || 'https://docs.sentry.io/';
  }, []);
```

Doing stuff like that directly inside the render function is a bit of an antipattern.